src/globalwarmingpotentials/__init__.py

```diff
@@ -9,370 +9,306 @@
-    "C3F8": 8900.0,
-    "cC4F8": 9540.0,
-    "C4F10": 9200.0,
-    "C5F12": 8550.0,
-    "C6F14": 7910.0,
-    "C7F16": 7820.0,
-    "C8F18": 7620.0,
-    "C10F18": 7190.0,
-    "SF5CF3": 17400.0,
-    "cC3F6": 9200.0,
-    "HFE125": 12400.0,
-    "HFE134": 5560.0,
-    "HFE143a": 523.0,
-    "HCFE235da2": 491.0,
-    "HFE245cb2": 654.0,
-    "HFE245fa2": 812.0,
-    "HFE347mcc3": 530.0,
-    "HFE347pcf2": 889.0,
-    "HFE356pcc3": 413.0,
-    "HFE449sl": 421.0,
-    "HFE569sf2": 57.0,
-    "HFE4310pccc124": 2820.0,
-    "HFE236ca12": 5350.0,
-    "HFE338pcc13": 2910.0,
-    "HFE227ea": 6450.0,
-    "HFE236ea2": 1790.0,
-    "HFE236fa": 979.0,
-    "HFE245fa1": 828.0,
-    "HFE263fb2": 1.0,
-    "HFE329mcc2": 3070.0,
-    "HFE338mcf2": 929.0,
-    "HFE347mcf2": 854.0,
-    "HFE356mec3": 387.0,
-    "HFE356pcf2": 719.0,
-    "HFE356pcf3": 446.0,
-    "HFE365mcf3": 1.0,
-    "HFE374pc2": 627.0,
-    "PFPMIE": 9710.0,
-    "CHCl3": 16.0,
-    "CH2Cl2": 9.0,
-    "CH3Cl": 12.0,
-    "Halon1201": 376.0,
-}
 
-AR5CCFGWP100 = {
-    "CH4": 34,
-    "N2O": 298,
-    "CFC11": 5352,
-    "CFC12": 11547,
-    "CFC13": 15451,
-    "CFC113": 6586,
-    "CFC114": 9615,
-    "CFC115": 8516,
-    "Halon1301": 7154,
-    "Halon1211": 2070,
-    "Halon2402": 1734,
-    "Halon1202": 280,
-    "CCl4": 2019,
-    "CH3Br": 3,
-    "CH3CCl3": 193,
-    "HCFC21": 179,
-    "HCFC22": 2106,
-    "HCFC123": 96,
-    "HCFC124": 635,
-    "HCFC141b": 938,
-    "HCFC142b": 2345,
-    "HCFC225ca": 155,
-    "HCFC225cb": 633,
-    "HFC23": 13856,
-    "HFC32": 817,
-    "HFC41": 141,
-    "HFC125": 3691,
-    "HFC134": 1337,
-    "HFC134a": 1549,
-    "HFC143": 397,
-    "HFC143a": 5508,
-    "HFC152": 20,
-    "HFC152a": 167,
-    "HFC161": 4,
-    "HFC227ea": 3860,
-    "HFC236cb": 1438,
-    "HFC236ea": 1596,
-    "HFC236fa": 8998,
-    "HFC245ca": 863,
-    "HFC245fa": 1032,
-    "HFC365mfc": 966,
-    "HFC4310mee": 1952,
-    "SO2F2": 4732,
-    "SF6": 26087,
-    "NF3": 17885,
-    "CF4": 7349,
-    "C2F6": 12340,
-    "C3F8": 9878,
-    "cC4F8": 10592,
-    "C4F10": 10213,
-    "C5F12": 9484,
-    "C6F14": 8780,
-    "C7F16": 8681,
-    "C8F18": 8456,
-    "C10F18": 7977,
-    "SF5CF3": 19396,
-    "cC3F6": 10208,
-    "HFE125": 13951,
-    "HFE134": 6512,
-    "HFE143a": 632,
-    "HCFE235da2": 595,
-    "HFE245cb2": 790,
-    "HFE245fa2": 981,
-    "HFE347mcc3": 641,
-    "HFE347pcf2": 1072,
-    "HFE356pcc3": 500,
-    "HFE449sl": 509,
-    "HFE569sf2": 69,
-    "HFE4310pccc124": 3353,
-    "HFE236ca12": 6260,
-    "HFE338pcc13": 3466,
-    "HFE227ea": 7377,
-    "HFE236ea2": 2143,
-    "HFE236fa": 1177,
-    "HFE245fa1": 997,
-    "HFE263fb2": 2,
-    "HFE329mcc2": 3598,
-    "HFE338mcf2": 1118,
-    "HFE347mcf2": 1028,
-    "HFE356mec3": 468,
-    "HFE356pcf2": 867,
-    "HFE356pcf3": 540,
-    "HFE365mcf3": 1,
-    "HFE374pc2": 758,
-    "PFPMIE": 10789,
-    "CHCl3": 20,
-    "CH2Cl2": 11,
-    "CH3Cl": 15,
-    "Halon1201": 454,
+globalwarmingpotentials = {
```

Comment:
This would mean that the API is
```python
globalwarmingpotentials.globalwarmingpotentials
```
which is not quite elegant. Hence the suggestion for a shorter name...

```suggestion
data = {
```

Reply:
This way, the nested dict could be retrieved as
```python
import globalwarmingpotentials as gwp
gwp.data
```

Reply:
Thanks for the feedback! :-)

Still more to type, but how about 
```py
from globalwarmingpotentials import globalwarmingpotentials as gwp

gwp["AR5GWP100"]
```

Reply:
Well, having a package and an attribute of the same name could be confusing, but I'll not veto the PR over it...

Reply:
I think it's the case for some (older?) Python packages, but i agree that it could be confusing :-)